compiler/one-cmds/one-codegen

```diff
@@ -68,18 +69,28 @@ def _verify_arg(parser, args):
 
 
 def _parse_arg(parser):
-    args, unknown_args = parser.parse_known_args()
+    codegen_args = []
+    backend_args = []
+    # delete file name
+    del sys.argv[0]
```

Comment:
I think instead of delete sys.argv[0] and use it, declare a copy of local argv from [1] and using it would be better.
I don't know how this is implemented in python but seems not comforatable...